easybuild/easyconfigs/k/Kaiju/Kaiju-1.7.3-gompi-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,50 @@
+# Updated from previous easyblock
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'Kaiju'
+version = '1.7.3'
+versionsuffix = '-Python-%(pyver)s'
+
+# invalid HTTPS cert
+homepage = 'http://kaiju.binf.ku.dk/'
+description = """Kaiju is a program for sensitive taxonomic classification of high-throughput
+sequencing reads from metagenomic whole genome sequencing experiments"""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'cstd': 'c++11'}
+
+# https://github.com/bioinformatics-centre/kaiju/archive/
+github_account = 'bioinformatics-centre'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.7.2-makefile.patch']
+checksums = [
+    '174ab6b6841d3d9164ec06f76a219a391d461d271b4a00fe8cf9cd87e689b05e',  # v1.7.3.tar.gz
+    '910e6671635fa7e23449aec8fbc4c07d7a48151fc5853a3f9ff4aab95ca9748b',  # Kaiju-1.7.2-makefile.patch
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Perl', '5.30.0'),
+]
+
+start_dir = 'src'
+
+files_to_copy = ['bin']
+
+sanity_check_paths = {
+    'files': [
+        'bin/%s' % x for x in [
+            'kaiju', 'kaiju2krona', 'kaiju2table', 'kaiju-addTaxonNames',
+            'kaiju-convertMAR.py', 'kaiju-convertNR', 'kaiju-gbk2faa.pl',
+            'kaiju-makedb', 'kaiju-mergeOutputs', 'kaiju-mkbwt', 'kaiju-mkfmi',
+            'kaijup', 'kaijux'
+        ]
+    ],
```

Comment:
This part could be nicely compressed as follows
```
    'files': ['bin/%s' % x for x in ['kaiju', 'kaiju2krona', 'kaiju2table', 'kaiju-addTaxonNames',
                                     'kaiju-convertMAR.py', 'kaiju-convertNR', 'kaiju-gbk2faa.pl', 'kaiju-makedb',
                                     'kaiju-mergeOutputs', 'kaiju-mkbwt', 'kaiju-mkfmi', 'kaijup', 'kaijux']],
```

Reply:
ok, we can compress even more :)

Reply:
nice :+1: